models/aether-2.0.x/yang/onf-device.yang

```diff
@@ -54,6 +54,21 @@ submodule onf-device {
         description
           "Link to simcard";
       }
+
+      container state {
+        config false;
+        description "read only attributes of device";
+
+        leaf ip-address {
+          description "the ip address assigned to the device";
+          type string;
```

Comment:
I did not tightly define the type here because it is a read only attribute - the data will always be created through our own software, and so will not be validated